.packit.yaml

```diff
@@ -99,6 +99,7 @@ jobs:
     targets:
       - fedora-latest-stable
       - fedora-development
+      - centos-stream-10-x86_64
```

Comment:
hmm, maybe this is better off as a separate set of jobs that uses `packages: [podman-centos]` .

Reply:
Ah, can do. It's quite some duplication, but maybe it magically helps with this [package install failure](https://artifacts.dev.testing-farm.io/680b1d9b-b191-4271-93eb-34fb79198b98/). The [log](https://artifacts.dev.testing-farm.io/680b1d9b-b191-4271-93eb-34fb79198b98/guest-setup-8ae95d92-7a98-48cd-a4cd-3545d64c29be/artifact-installation-8ae95d92-7a98-48cd-a4cd-3545d64c29be/4-Install-packages.txt) shows some qemu package conflicts.

Reply:
the logs show it was trying to fetch from `fedora-10` repos. Hopefully the extra set of jobs should work, but testing-farm/packit seems to be slow for centos-stream on this PR.